File: equivalence-partitioning/generate-tc-ep.rb
value_for_a = [(-200..0), (1..200), (201..300)]
value_for_b = [(-200..0), (1..200), (201..300)]
value_for_c = [(-200..0), (1..200), (201..300)]

value_for_a.each do |a_arr|
  value_for_b.each do |b_arr|
    value_for_c.each do |c_arr|
      a = a_arr.to_a.sample
      b = b_arr.to_a.sample
      c = c_arr.to_a.sample
      puts "#{a}, #{b}, #{c}"
    end
  end
end

File: equivalence-partitioning/generate-tc-ep.py
import random

value_for_a = [range(-200, 0), range(1,200), range(201,300)]
value_for_b = [range(-200, 0), range(1,200), range(201,300)]
value_for_c = [range(-200, 0), range(1,200), range(201,300)]

for equivalence_part_a in value_for_a:
  for equivalence_part_b in value_for_b:
    for equivalence_part_c in value_for_c:
      a = random.choice(list(equivalence_part_a))
      b = random.choice(list(equivalence_part_b))
      c = random.choice(list(equivalence_part_c))
      print("%d,  %d,  %d" % (a, b, c))
      